### PR TITLE
Scan released directory for images

### PR DESCRIPTION
**Pull request by thedadams**
Issue:
https://github.com/rancher/rancher/issues/31995

Original PR:
https://github.com/rancher/rancher/pull/32100

### CODE REVIEW

**Review by cbron — DISMISSED** [review]
lgtm, but wait for reviews on other pr to merge

File: scripts/package [hunk 1]
```diff
@@ -53,11 +53,13 @@ if [ ! -d build/charts ]; then
...
         TGZ_REL_PATH=$CHART_REPO_DIR/$TGZ_PATH
+        TGZ_EXTRACT_PATH=$(dirname $CHART_REPO_DIR/${TGZ_PATH##released/})
         if [[ $TGZ_PATH == *crd*.tgz ]]; then
             echo "Skipped CRD: $TGZ_REL_PATH"
         else
-            echo "Extract: $TGZ_REL_PATH"
-            tar -xvf $TGZ_REL_PATH -C $(dirname $TGZ_REL_PATH)
+            echo "Extract: $TGZ_REL_PATH to $TGZ_EXTRACT_PATH"
+            mkdir -p $TGZ_EXTRACT_PATH
+            tar -xvf $TGZ_REL_PATH -C $TGZ_EXTRACT_PATH
```
Comment (cbron) [inline scripts/package:62]: So in short: 
* on release branches tgz's get unfurled in the same dir
* on dev branches a `./assets` dir is made (from `./released/assets`) and  tgz's get unfurled in there

Comment (StrongMonkey) [reply scripts/package:62]: Is it just replace the tgz url so that `./released/assets/x.y.z.00.tgz` -> `./assets/x.y.z.00.tgz`?

Comment (thedadams) [reply scripts/package:62]: For release branches, everything gets extracted to the `assets` directory.

For dev branches, the charts are in the `released` directory, but get extracted to the `assets` directory.

Comment (cbron) [reply scripts/package:62]: And in both cases the `/assets` dir gets passed to image export.

**Review by cbron — APPROVED** [review]

**Review by cbron — APPROVED** [review]
lgtm

**Pull request merged**